src/utils/getSocketIntegration.ts

```diff
@@ -1,4 +1,4 @@
-export type IntegrationType = 'wds';
+export type IntegrationType = 'wds' | 'whm' | (string & {});
```

Comment:
this allows to pass any string while still getting autocomplete by typescript for the `'wds'` and `'whm'` options.